walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -70,4 +62,18 @@
 
     return answers;
   }
+
+  private List<TimeRange> getRelatedEvents(Collection<Event> events, Collection<String> attendees) {
```

Comment:
Small nitpick: this function can be static

Reply:
oh yeah =') but somehow I feel the need to make it private as it's a child function. any disadvantage in mind?

Reply:
You can make it private static.